src/common/utility.js

```diff
@@ -182,3 +188,30 @@ export const nonENLangUrlReplace = (current_path) => {
     const path_with_or_without_slash = /\/.+?(\/)|(\/[a-zA-Z'-]+)/u
     return current_path.replace(path_with_or_without_slash, '')
 }
+
+export const getLiveChatStorage = () =>
+    isBrowser() ? localStorage.getItem('live_chat_redirection') : null
+
+export const removeLocalStorage = (prop) => localStorage.removeItem(prop)
+
+export const getLiveChatRedirectStatus = (lang_status) => {
+    const lang = getLanguage()
+    const live_chat_enable = getLiveChatStorage()
+
+    if (lang_status && live_chat_enable) {
+        return true
+    } else if (lang == 'en' && live_chat_enable) {
+        return true
+    }
```

Comment:
```suggestion
    return (lang_status && live_chat_enable) || (lang == 'en' && live_chat_enable);
```

Reply:
Im looking for this actually..thanks @yashim-deriv ..amended in the latest commit